suite-native/atoms/src/Sheet/BottomSheet.tsx

```diff
@@ -48,6 +49,7 @@ export const BottomSheet = ({
     subtitle,
     children,
     isScrollable = true,
+    footer = undefined,
```

Comment:
nit: Unnecessary default value of `undefined`. The optional props are `undefined` even without explicit assignment. 
```suggestion
    footer,
```

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/47c932e7e017d515a6346a59d9ceca1f2d6292cd.